tests/acceptance/blocks/thememap.js

```diff
@@ -29,6 +38,58 @@ class ThemeMap {
   async clickSearchThisAreaButton () {
     await t.click(this._searchThisAreaButton);
   }
+
+  async clickMap () {
+    await t.wait(10000);
+    await t.click(this._canvas);
+    await t.wait(10000);
+  }
+
+  /**
+   * Gets the current zoom of the map
+   * @returns {Promise<number>}
+   */
+  async getZoom () {
+    const zoom = await ClientFunction(() => {
+      return ANSWERS
+        .components
```

Comment:
does `ANSWERS.components.getActiveComponent()` work here?

Reply:
would that be less robust? If we get the component by name we know for sure that it is the VerticalFullPageMapOrchestrator. If we use getActiveComponent(), would that change baseed on which components were used before calling `getZoom()`?

Reply:
getActiveComponent() gets the first component by type, and here we get the first component by name, so I think it should be roughly the same robustness (unless I'm misunderstanding something)?

Reply:
oh, that makes sense. I hadn't used `getActiveComponent()` before and I didn't realize it accepted a parameter